docs/source/index.rst

```diff
@@ -0,0 +1,23 @@
+================================
+Home Assistant API Documentation
+================================
+
+Public API documentation for `Home Assistant developers`_.
+
+Contents:
+
+.. toctree::
+   :maxdepth: 2
+
+   api/core
+   api/entity
+   api/event
```

Comment:
If we go with wildcards and `:glob:` here, there will be no need to update the tree for new entries.